wandb/docker/__init__.py

```diff
@@ -216,9 +212,9 @@ def get_image_uid(image_name: str) -> int:
     return int(shell(["run", image_name, "id", "-u"]))
 
 
-def push(repo: str, tag: str) -> Optional[str]:
+def push(image: str, tag: str) -> Optional[str]:
     """Push an image to a remote registry"""
-    return shell(["push", f"{repo}:{tag}"])
+    return shell(["push", f"{image}:{tag}"])
```

Comment:
I think this should be repo. In the case of docker hub and AWS ECR at least this represents the repo name